File: server/iih_pub/iih.ci_pub/src/main/java/iih/ci/ord/dto/ordertemplatedto/d/OrderTemplateDTO.java
package iih.ci.ord.dto.ordertemplatedto.d;

import xap.mw.core.data.*;
import xap.mw.coreitf.d.*;
import java.math.BigDecimal;

/**
 * 助手医嘱模板DTO DTO数据 
 * 
 */
public class OrderTemplateDTO extends BaseDTO {
	private static final long serialVersionUID = 1L;
	
	/**
	 * 主键
	 * @return String
	 */
	public String getId_template() {
		return ((String) getAttrVal("Id_template"));
	}
	/**
	 * 主键
	 * @param Id_template
	 */
	public void setId_template(String Id_template) {
		setAttrVal("Id_template", Id_template);
	}
	/**
	 * 医嘱模板
	 * @return FMap
	 */
	public FMap getSrvortplitemaggdo() {
		return ((FMap) getAttrVal("Srvortplitemaggdo"));
	}
	/**
	 * 医嘱模板
	 * @param Srvortplitemaggdo
	 */
	public void setSrvortplitemaggdo(FMap Srvortplitemaggdo) {
		setAttrVal("Srvortplitemaggdo", Srvortplitemaggdo);
	}
	/**
	 * 频次
	 * @return FMap
	 */
	public FMap getFreqdefdo() {
		return ((FMap) getAttrVal("Freqdefdo"));
	}
	/**
	 * 频次
	 * @param Freqdefdo
	 */
	public void setFreqdefdo(FMap Freqdefdo) {
		setAttrVal("Freqdefdo", Freqdefdo);
	}
	/**
	 * 剂量单位
	 * @return FMap
	 */
	public FMap getMeasdocdo() {
		return ((FMap) getAttrVal("Measdocdo"));
	}
	/**
	 * 剂量单位
	 * @param Measdocdo
	 */
	public void setMeasdocdo(FMap Measdocdo) {
		setAttrVal("Measdocdo", Measdocdo);
	}
}

File: client/iih_stub/iih.ci_stub/iih.ci.ord/dto/ordertemplatedto/d/OrderTemplateDTO.cs
using System;
using xap.mw.core.data;
using xap.mw.coreitf.d;

namespace iih.ci.ord.dto.ordertemplatedto.d
{
    /// <summary>
    /// OrderTemplateDTO 的摘要说明。
    /// </summary>
    public class OrderTemplateDTO : BaseDTO {

        public OrderTemplateDTO() {
 
        }

        /// <summary>
        /// 主键
        /// </summary>
		public string Id_template {
            get { return getAttrVal<string>("Id_template",null); }
            set { setAttrVal<string>("Id_template", value); }
        }

        /// <summary>
        /// 医嘱模板
        /// </summary>
		public FMap Srvortplitemaggdo {
            get { return getAttrVal<FMap>("Srvortplitemaggdo",null); }
            set { setAttrVal<FMap>("Srvortplitemaggdo", value); }
        }

        /// <summary>
        /// 频次
        /// </summary>
		public FMap Freqdefdo {
            get { return getAttrVal<FMap>("Freqdefdo",null); }
            set { setAttrVal<FMap>("Freqdefdo", value); }
        }

        /// <summary>
        /// 剂量单位
        /// </summary>
		public FMap Measdocdo {
            get { return getAttrVal<FMap>("Measdocdo",null); }
            set { setAttrVal<FMap>("Measdocdo", value); }
        }
        /// <summary>
        /// 返回属性列表
        /// </summary>
        /// <returns></returns>
        public override string[] getAttrNames()
        {
            return new string[]{ "Id_template", "Srvortplitemaggdo", "Freqdefdo", "Measdocdo"};
        }
        
        /// <summary>
        /// 返回全路径DO类名
        /// </summary>
        /// <returns></returns>
        public override string getFullClassName()
        {
            return "iih.ci.ord.dto.ordertemplatedto.d.OrderTemplateDTO";
        }
    }
}
